### ISSUE TITLE
'ipam.models.MultipleObjectsReturned'

### ISSUE THREAD

**Issue by Fyzzle** [digitalocean/netbox#719]
I had to migrate the machine from a Hyper-V environment to an ESX one, and the V2V didn't like the Ubuntu 16.04 server I had it on so...

I fired up a new server, went through the setup process and restored the database using:

pg_dumpall -h (ip of server) > backup.bak
psql -f backup.bak postgres

Pretty much everything works except a few subnets under Prefixes. On those I get the error:

>  Server Error
> There was a problem with your request. This error has been logged and administrative staff have been notified. Please return to the home page and try again.
> 
> If you are responsible for this installation, please consider filing a bug report. Additional information is provided below:
> 
> <class 'ipam.models.MultipleObjectsReturned'>
> 
> get() returned more than one Aggregate -- it returned 2!

I'm guessing I have a duplicate entry somewhere, but I'm not sure how to track down and delete the funky one, or if there's a better way to do it.

**Comment by jeremystretch** [comment]
That's odd. Does the list of aggregates in the admin UI (IPAM > Aggregates) look correct? You can also check via the shell by running `./manage.py shell` and pasting the following code:

```
from ipam.models import Aggregate
for a in Aggregate.objects.all():
    print('[{}] {}'.format(a.pk, a))
```

**Comment by Fyzzle** [comment]
Yeah it looks great, both in the web interface and from the shell.

**Comment by Fyzzle** [comment]
Wait I figured it out, I'm a dummy. Posting fix.

**Comment by Fyzzle** [comment]
Sorry everyone, false alarm.  Can I still blame a turkey hangover?

**Comment by jeremystretch** [comment]
@Fyzzle I'm curious how you ended up with overlapping aggregates in the first place, though. There's validation in place that's supposed to prevent that from happening.

**Comment by Fyzzle** [comment]
Ah gotcha, I'm a little overextended today but everything is fresh in my head.  I'll write up a step by step of what I did.

**Comment by Fyzzle** [comment]
Okie dokie:

I went through the docs and installed on Ubuntu 16.04 server using the Debian instructions.

I installed by cloning the git repo and went step by step through the rest up until the "Run Database Migrations".  _Before_ that step I restored the database to the new server.  I think that's where the hang up was, I should have done it later in the process.  

The rest of the install was done as written, I did not skip any steps in the process.
